api/src/wfl/environment.clj

```diff
@@ -1,28 +1,27 @@
 (ns wfl.environment
   "Map environment to various values here."
-  (:require [clojure.data.json  :as json]
+  (:require [clojure.data.json :as json]
             [clojure.tools.logging :as log]
             [clojure.string :as str]
-            [vault.client.http]         ; vault.core needs this
-            [vault.core :as vault]
-            [wfl.util  :as util]))
+            [vault.client.http] ; vault.core needs this
+            [vault.core :as vault]))
+
+(declare getenv)
```

Comment:
Please move vault-secrets instead.
And please remove the `is-known-cromwell-url?` TODO also.
And `DO NOT USE THIS`?

Reply:
vault secrets depends on getenv.

Reply:
And `DO NOT USE THIS` is so that people don't use `testing` to hack stuff.

Reply:
That's because System/getenv -=> wfl.environment/getenv which I wouldn't do, but whatever.

Reply:
> And `DO NOT USE THIS` is so that people don't use `testing` to hack stuff.

Then howzabout?

  "Override the environment used by `getenv` for testing. Use
  `wfl.tools.fixtures/with-temporary-environment` instead of this."

Reply:
That's exactly the same this witj `DO NOT USE THIS` removed?